graphql.go

```diff
@@ -187,20 +188,21 @@ func (c *Client) request(ctx context.Context, query string, variables map[string
 		if c.debug {
 			err = err.withRequest(request, reqReader)
 		}
-		return nil, nil, nil, Errors{err}
+		return nil, nil, nil, nil, Errors{err}
 	}
 
 	var out struct {
-		Data   *json.RawMessage
-		Errors Errors
+		Data       *json.RawMessage
+		Extensions map[string]*json.RawMessage
```

Comment:
```suggestion
		Extensions json.RawMessage
```

IMO it's better to make `extensions` as a `json.RawMessage`. Users will define a map themselves

Reply:
sure, changed to json.RawMessage